src/foam/core/AbstractArrayPropertyInfo.java

```diff
@@ -21,7 +22,42 @@
 {
   @Override
   public void setFromString(Object obj, String value) {
-    // TODO
+    if ( value == null ) {
+      this.set(obj, null);
+      return;
+    }
+    java.util.List<String> list = new java.util.LinkedList<String>();
```

Comment:
Import so you can just refer to java.util.List as List. Same for LinkedList. Add a TODO: to reuse this list with a ThreadLocal for better performance.